librz/bin/format/mach0/mach0.c

```diff
@@ -8,12 +8,14 @@
 #include "mach0.h"
 #include <rz_msg_digest.h>
 
+#include "mach0_utils.inc"
+
 // TODO: deprecate bprintf and use Eprintf (bin->self)
```

Comment:
Maybe change the comment to `Deprecate bprintf and eprintf and use RZ_LOG_*() functions instead`?